README.md

```diff
@@ -79,6 +84,17 @@ export class DemoApp {
        [options]="options"
        (onUpload)="handleUpload($event)">
 
+<!-- drag & drop file example-->
+<style>
+  .file-over { border: dotted 3px red; } /* Default class applied to drop zones on over */
+</style>
+<div ng2FileDrop
```

Comment:
typo. directive is named `ngFileDrop` not `ng2FileDrop` so this example would not work :) please fix.